src/lib/notcurses.c

```diff
@@ -590,14 +614,14 @@ ncplane* ncplane_new_internal(notcurses* nc, ncplane* n,
     p->leny = nopts->rows;
     p->lenx = nopts->cols;
   }
-  size_t fbsize = sizeof(*p->fb) * (p->leny * p->lenx);
-  if((p->fb = malloc(fbsize)) == NULL){
+
+  size_t fbsize = ncplane_sizeof_cellarray( p->leny, p->lenx);
+  if( ! fbsize || (p->fb = calloc(1,fbsize)) == NULL){
```

Comment:
why the move from `malloc()` to `calloc()`? oh, to eliminate the `memset()`? gotcha. that's reasonable.

Reply:
standard style for this project would be

`if(!fbsize || ((p->fb = calloc(1, fbsize)) == NULL)){

Reply:
Because `calloc` can and will (for large allocations) return already zero filled memory by the OS on Linux/BSD/macOS.  Also you don't have to page them all in for the superflous memset.

Reply:
hmmm, i would have hoped the compiler could work that out, but i suppose it requires `mmap()` knowledge